src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceCreationService.kt

```diff
@@ -147,6 +148,7 @@ class LicenceCreationService(
     val licence = if (isTimeServedLogicEnabled && hardStopKind == LicenceKind.TIME_SERVED) {
       LicenceFactory.createTimeServe(
         licenceType = licenceType,
+        eligibleKind = cvlRecord.eligibleKind,
```

Comment:
I think this will currently break with any TIME_SERVED licence being created (eg by an API call for testing). the `eligibleKind` will be either CRD or PRRD, and the model only lists `TIME_SERVED` as an allowable value.

I'm not familiar enough with `allowableValue` to know if this will actually throw an error or not in practice.